reconfiguration/cmf/concord.cmf

```diff
@@ -192,12 +193,19 @@ Msg ClientKeyExchangeStatusResponse 48 {
      list kvpair uint32 ClientExchangePublicKey clients_keys
 }
 
+Msg ClientsAddRemoveExecuteCommand 51 {
+    string config_descriptor
+    string token
+    bool restart
+}
+
 Msg ClientStateReply 39 {
     uint64 block_id
     oneof {
         ClientExchangePublicKey
         ClientKeyExchangeCommand
         ClientsAddRemoveCommand
+        ClientsAddRemoveExecuteCommand
```

Comment:
By adding this new message, we cannot return the client the "ClientsAddRemoveCommand" as it contains the tokens of all the clients.
I would suggest to either remove the ClientsAddRemoveCommand from the possible responses to the CRE or to assume that we send the clients' addRemove commands one by one

Reply:
Fixed